app/Helpers/database/player-game.php

```diff
@@ -187,6 +187,10 @@ function getUserProgress(string $user, array $gameIDs, int $numRecentAchievement
         $libraryOut['GameInfo'] = $gameInfo;
     }
 
+    // magic numbers!
+    // -1 = don't populate RecentAchievements field
+    // 0 = return all achievements for each game, with the unlocked achievements first ordered by unlock date
+    // >0 = return the N most recent unlocks across all games queried, grouped by game
```

Comment:
-1 when called from API_GetUserProgress and API_GetUserRecentlyPlayedGames
0 when called from the user profile page, which needs the unlocked achievements
>0 when called from API_GetUserSummary, which shouldn't return the unlocked achievements